securesignals-ima-devapp/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.uid2.securesignals.ima.devapp'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.uid2.securesignals.ima.devapp"
+        minSdk 26
```

Comment:
Can I ask why the high `minSdk` here? The UID2 dev-app uses `21`. I'm curious what in the IMA (or IMA/Dev-App) requires the bump? If nothing, maybe we should drop to `21` to be consistent.

Reply:
I've refactored this to use `System.currentTimeMillis()` so that the API could drop down to 21.